pkgs/by-name/kb/kbld/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kbld";
+  version = "0.44.1";
+
+  src = fetchFromGitHub {
+    owner = "carvel-dev";
+    repo = "kbld";
+    rev = "v${version}";
+    hash = "sha256-sEzCA32r3nSY1hT1r4EPPWsF9Kgn0rXnaAKlatFjZIo=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [ "cmd/kbld" ];
+
+  CGO_ENABLED = "0";
+  ldflags = [
+    "-X=carvel.dev/kbld/pkg/kbld/version.Version=${version}"
+  ];
+
+  meta = {
+    description = "Kbld seamlessly incorporates image building and image pushing into your development and deployment workflows";
```

```suggestion
    description = "Seamlessly incorporates image building and image pushing into your development and deployment workflows";
```

Don't repeat package name